packages/clima_ui/lib/themes/black_theme.dart

```diff
@@ -12,6 +12,7 @@ ThemeData blackTheme = ThemeData.dark().copyWith(
   dialogBackgroundColor: const Color(0xFF202125),
   toggleableActiveColor: const Color(0xFF89B4F8),
   accentColor: const Color(0xFF89B4F8),
+  colorScheme: ColorScheme.fromSwatch(accentColor: const Color(0xFF89B4F8)),
```

Comment:
Is the `accentColor` thing above this line necessary anymore after adding this?

Also, see https://api.flutter.dev/flutter/material/ThemeData/colorScheme.html:
> New components can be defined exclusively in terms of colorScheme. Existing components will gradually migrate to it, to the extent that is possible without significant backwards compatibility breaks.

We can think later about removing as much of the specific properties as possible in favor of `colorScheme` if it doesn't result in worse colors.

Reply:
I am not sure honestly, but I think you are right

Reply:
We can decide on this later I guess